mobify/platform-scaffold
# WEB-1531: Add Meta Data to Account Pages (SEO)

Add SEO helpful `<meta>` and `<title>` tags to the site header via the [`react-helmet`](https://www.npmjs.com/package/react-helmet) dependency.

**JIRA**: [WEB-1531](https://mobify.atlassian.net/browse/WEB-1531)

**Linked PRs**: #1009, and [progressive-web-sdk#866](https://github.com/mobify/progressive-web-sdk/pull/866)

## Changes
- New `pageMeta` key added to the following pages/containers:
    - Login page
    - Registration page
    - Account Dashboard
    - Account Information page
    - Account Address Book page
    - Account My Orders page
    - Account Order pages ("view order" for each individual order page)
    - Account Wishlist page

## Todos
- [x] Merge [progressive-web-sdk#866](https://github.com/mobify/progressive-web-sdk/pull/866)
- [ ] Merge #1009
- ~~(if applicable) analytics events instrumented to measure impact of change~~
- Add `PageMeta` to more pages in future PRs
- [ ] Update SDK to latest version that includes the required SEO updates (see related PRs)

## How to test-drive this PR
**IMPORTANT:** run `npm i` to install new dependency
- Link the SDK to the scaffold
    - cd to the SDK repository
    - run `git checkout WEB-1531__more-pages-meta`
    - run `npm run prod:build`
    - run `npm link`
    - return to the scaffold by cd'ing to the `platform-scaffold` repository
    - `npm i` for `react-helmet` if you haven't done this yet
- Run `npm start`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Visit any of the affected pages (as listed above)
- **Verify** that the site `title` updates appropriately
- **Verify** that a `<meta name="description">` tag is present on that pages and matches what's on desktop
- **Verify** that a `<meta name="keywords">` tag is present on that pages and matches what's on desktop (in the case of Merlin's it will be empty)
- _Repeat above steps with the SFCC connector_
